src/patternfly/components/Table/table-toggle-icon.hbs

```diff
@@ -1 +1,6 @@
-<i class="fas fa-angle-down pf-c-table__toggle-icon" aria-hidden="true"></i>
\ No newline at end of file
+<span class="pf-c-table__toggle-icon{{#if table-toggle-icon--modifier}} {{table-toggle-icon--modifier}}{{/if}}"
```

Comment:
need to update the docs for `.pf-c-table__toggle-icon` to reflect the element change